src/components/share-structure/EditShareStructure.vue

```diff
@@ -351,21 +356,22 @@ export default class EditShareStructure extends Mixins(CurrencyLookupMixin) {
   }
 
   /** The rules applying to the Class/Series par value input field. */
-  get parValueRule (): Array<(v) => boolean | string> {
-    let rules: Array<(v) => boolean | string> = []
+  get parValueRule (): Array<VuetifyRuleFunction> {
+    let rules: Array<VuetifyRuleFunction> = []
     if (!this.hasNoParValue) {
       rules = [
-        (v: string) => (v !== '' && v !== null && v !== undefined) || 'Par value is required',
```

Comment:
I don't need to declare that v is a string because the v-text-field component is already typed as a number.